<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>360</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>675</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>675</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QSORDER</string>
  </property>
  <property name="windowIcon">
   <iconset resource="bkg_img.qrc">
    <normaloff>:/logo/qsorder.png</normaloff>:/logo/qsorder.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>0.970000000000000</double>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>675</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(243, 239, 222);
background-image: url(:/bkground/ricepaper_v3.png);</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="console_tab">
    <attribute name="title">
     <string>Qsorder Console</string>
    </attribute>
    <widget class="QPlainTextEdit" name="console">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>674</width>
       <height>231</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px;</string>
     </property>
     <property name="tabChangesFocus">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string/>
     </property>
     <property name="backgroundVisible">
      <bool>false</bool>
     </property>
     <property name="centerOnScroll">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_input">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>251</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>input input input input input in</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>308</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>UDP Port:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>251</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Rec. Input:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_delay">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>270</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>55</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>250</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Buffer, secs:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_port">
     <property name="geometry">
      <rect>
       <x>129</x>
       <y>308</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>port</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>270</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Delay, secs:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_buffer">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>250</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>555</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>290</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Qs in buffer:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_queue">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>290</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>55</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>310</y>
       <width>70</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Qsorder ver:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_version">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>310</y>
       <width>18</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>5.5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="quitButton_2">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>270</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="manual_dump_btn">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>270</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Audio Buffer</string>
     </property>
     <property name="shortcut">
      <string>Ctrl+Alt+O</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>240</y>
       <width>20</width>
       <height>81</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>641</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>270</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Qsorder status:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_status">
     <property name="geometry">
      <rect>
       <x>127</x>
       <y>270</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Not Running</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>289</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Dropbox status:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_dropbox_status">
     <property name="geometry">
      <rect>
       <x>127</x>
       <y>289</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Not configured</string>
     </property>
    </widget>
    <zorder>label_21</zorder>
    <zorder>label_status</zorder>
    <zorder>label_23</zorder>
    <zorder>label_dropbox_status</zorder>
    <zorder>line_2</zorder>
    <zorder>console</zorder>
    <zorder>label_input</zorder>
    <zorder>label_15</zorder>
    <zorder>label_4</zorder>
    <zorder>label_port</zorder>
    <zorder>label_19</zorder>
    <zorder>label_queue</zorder>
    <zorder>label_6</zorder>
    <zorder>label_buffer</zorder>
    <zorder>label_delay</zorder>
    <zorder>label_14</zorder>
    <zorder>label_25</zorder>
    <zorder>label_version</zorder>
    <zorder>quitButton_2</zorder>
    <zorder>manual_dump_btn</zorder>
    <zorder>line</zorder>
   </widget>
   <widget class="QWidget" name="config_tab">
    <attribute name="title">
     <string>Configuration</string>
    </attribute>
    <widget class="QCheckBox" name="debug">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Debug Log</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>281</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>QSORDER Configuration Parameters</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="continuous">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Continuous Recording</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="so2r">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>SO2R Mode</string>
     </property>
    </widget>
    <widget class="QComboBox" name="inputs">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>120</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Audio Input:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>210</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Settings</string>
     </property>
    </widget>
    <widget class="QPushButton" name="applyButton">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>210</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply w/o Saving</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>120</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Dropbox Storage Key:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="drop_key">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>120</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="getDropbox_btn">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>150</y>
       <width>172</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Obtain Dropbox Key</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>160</y>
       <width>131</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="bkg_img.qrc">:/logo/qsorder.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="quitButton">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>270</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>171</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Hot Key: CTRL+ALT+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="hotkey">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777205</height>
         </size>
        </property>
        <property name="text">
         <string>O</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>182</width>
       <height>54</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="1,1" columnstretch="2,1">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Buffer, secs:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="buffer">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>45</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Delay, secs:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="delay">
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>20</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>221</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Listen on UDP Port:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="port">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="minimum">
         <number>1024</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>12060</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QSpinBox" name="station_nr">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>50</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>1024</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>50</y>
       <width>100</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Station Number:</string>
     </property>
    </widget>
    <widget class="QToolButton" name="selectDir_btn">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>80</y>
       <width>30</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>80</y>
       <width>421</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Path:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="path">
        <property name="text">
         <string>&lt;current directory&gt;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <zorder>quitButton</zorder>
    <zorder>debug</zorder>
    <zorder>label_2</zorder>
    <zorder>continuous</zorder>
    <zorder>so2r</zorder>
    <zorder>inputs</zorder>
    <zorder>label_11</zorder>
    <zorder>saveButton</zorder>
    <zorder>applyButton</zorder>
    <zorder>label_12</zorder>
    <zorder>drop_key</zorder>
    <zorder>getDropbox_btn</zorder>
    <zorder>label_13</zorder>
    <zorder>layoutWidget</zorder>
    <zorder>layoutWidget</zorder>
    <zorder>layoutWidget</zorder>
    <zorder>layoutWidget</zorder>
    <zorder>station_nr</zorder>
    <zorder>label_10</zorder>
    <zorder>selectDir_btn</zorder>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="bkg_img.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>quitButton_2</sender>
   <signal>clicked()</signal>
   <receiver>quitButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>562</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>562</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>manual_dump()</slot>
 </slots>
</ui>
